packages/cursorless-engine/src/snippets/snippet.ts

```diff
@@ -54,6 +54,12 @@ export function transformSnippetVariables(
         candidate.parent.replace(candidate, [placeholder]);
       }
     }
+
+    if (candidate instanceof Placeholder) {
```

Comment:
Nitpicking here, but I believe this should be an else if

Reply:
I mean there is no way of both branches running so logically it will be the same result, but if you prefer that syntax it's fine for me.

Reply:
It's a tiny bit more efficient, and makes it clear they're mutually exclusive